### PR TITLE
Migrate legacy UI components content

### PR DESCRIPTION
**Pull request by nikhil-dabhade**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/ui-legacy/api/overview.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Overview
+description: Overview
+filterKey: framework
+---
+
+<inline-fragment framework="vue" src="~/ui-legacy/fragments/vue/overview.md"></inline-fragment>
```
Comment (jordanranz) [inline docs/ui-legacy/api/overview.md:7]: Should this also include a react overview?

File: docs/ui-legacy/auth/fragments/react/authenticator.md [hunk 2]
```diff
@@ -0,0 +1,390 @@
+## Using withAuthenticator HOC
+
+For React and React Native apps, the simplest way to add authentication flows into your app is to use the `withAuthenticator` Higher Order Component.
+
+If you are using the `withAuthenticator` HOC with React Native, make sure to link `react-native-vector-icons` and then (re)launch your app:
+
+```bash
+react-native link react-native-vector-icons
```
Comment (jordanranz) [inline docs/ui-legacy/auth/fragments/react/authenticator.md:8]: This and the following line can be removed

File: docs/ui-legacy/auth/fragments/react/authenticator.md [hunk 3]
```diff
@@ -0,0 +1,390 @@
+## Using withAuthenticator HOC
+
+For React and React Native apps, the simplest way to add authentication flows into your app is to use the `withAuthenticator` Higher Order Component.
+
+If you are using the `withAuthenticator` HOC with React Native, make sure to link `react-native-vector-icons` and then (re)launch your app:
```
Comment (jordanranz) [inline docs/ui-legacy/auth/fragments/react/authenticator.md:5]: This line can be removed

File: docs/ui-legacy/auth/fragments/react/authenticator.md [hunk 4]
```diff
@@ -0,0 +1,390 @@
+## Using withAuthenticator HOC
+
+For React and React Native apps, the simplest way to add authentication flows into your app is to use the `withAuthenticator` Higher Order Component.
```
Comment (jordanranz) [inline docs/ui-legacy/auth/fragments/react/authenticator.md:3]: We can make this just React

File: docs/ui-legacy/auth/fragments/angular/authenticator.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
+AWS Amplifies provides UI components that you can use in your view templates.
```
Comment (sammartinez) [inline docs/ui-legacy/auth/fragments/angular/authenticator.md:1]:
```suggestion
AWS Amplify provides UI components that you can use in your view templates. 
```

File: docs/ui-legacy/auth/fragments/react/sign-up-attributes.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
+<table class="doc-table">
```
Comment (jordanranz) [inline docs/ui-legacy/auth/fragments/react/sign-up-attributes.md:1]: Doesn't look like these tables will get style because this css is out of scope. We should migrate this to use md tables.

**Comment by jordanranz** [comment]
Overall this looks good, a few comments. We can chat offline and quickly go through this.

File: docs/ui-legacy/api/overview.md (hunk 1 above)
Comment (nikhil-dabhade) [reply docs/ui-legacy/api/overview.md:7]: Yes. I added it. We need to have net new content for overview for react as we have it for Vue and Angular.

File: docs/ui-legacy/auth/fragments/react/authenticator.md (hunk 4 above)
Comment (nikhil-dabhade) [reply docs/ui-legacy/auth/fragments/react/authenticator.md:3]: Done

File: docs/ui-legacy/auth/fragments/react/sign-up-attributes.md (hunk 6 above)
Comment (nikhil-dabhade) [reply docs/ui-legacy/auth/fragments/react/sign-up-attributes.md:1]: Correct. They don't get the consistent styling as we get using <ui-component-props>
Do we have a styling in md that would match the ones we have for the new ui library  ?

File: docs/ui-legacy/api/overview.md (hunk 1 above)
Comment (jordanranz) [reply docs/ui-legacy/api/overview.md:7]: Ok, this can just be a small overview blub we can add
